client/main.scss

```diff
@@ -38,7 +38,7 @@ h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
   min-height:calc(100vh - 70px - 65px);
 }
 
-html, body, #react-root {
+html, body, #react-root, [data-reactroot] {
```

Comment:
You need to specify an element into which to render the react app (which is in this case `#react-root`), but that then creates an other element immediately inside it with the `[data-reactroot]` attribute.

I did consider making the container `body`, but I think that's generally discouraged.

To be honest, the real problem is `height` not being passed to children of children. Grrr.